src/components/manage/History/History.jsx

```diff
@@ -145,16 +145,19 @@ class History extends Component {
           <Table selectable compact singleLine attached>
             <Table.Header>
               <Table.Row>
-                <Table.HeaderCell width={4}>
+                <Table.HeaderCell width={3}>
                   <FormattedMessage id="What" defaultMessage="What" />
                 </Table.HeaderCell>
-                <Table.HeaderCell width={4}>
+                <Table.HeaderCell width={3}>
+                  <FormattedMessage id="Version" defaultMessage="Version" />
```

Comment:
@iFlameing Maybe this should be "Version Number" of "Version #". Just thinking out loud here. I am undecided...other opinions? @tiberiuichim @avoinea @jackahl?

Reply:
I think "Version" is clear enough, because there are actual number below it so I guess it should be easy for users to understand. I would be ok with "Version #" if you prefer having some hint. I wouldn't make the column header wider than that with a longer string, since the rest of the column will always be filled by small numbers.

On the other hand, I would suggest using a more verbose id for the translation string, in order to avoid possible clashes in the future. For example:

* version_number
* Version Number
* History Version
* History Version Number

Reply:
In fact, being the version column so small, it could also be arguable to reduce its width to 2 and increase the What column to 4.

Reply:
+1 to the better translation strings.